app/code/community/TIG/MyParcel2014/Model/Api/MyParcel.php

```diff
@@ -274,6 +274,33 @@ public function getRequestErrorDetail()
         return false;
     }
 
+	/**
+	 *
+	 * Get the Megento version and MyParcel version
+	 *
+	 * @return $this
+	 */
+	protected function getUserAgent()
```

Comment:
Met **get**Function() return je bijna nooit een $this. Bij een get... verwacht ik meestal een waarde. Als je in ```$this->magento_version``` iets stopt dan moet je ```setUserAgent``` gebruikten. Dan set je iets ipv dat je get gebruikt.

In dit geval zou je ook getUserAgent kunnen houden. Dan set je niets met bijvoorbeeld ```$this->magento_version```, maar dan stuur je direct de berekening terug.